src/main/drivers/dma.c

```diff
@@ -63,23 +63,46 @@ DEFINE_DMA_IRQ_HANDLER(2, 4, DMA2_CH4_HANDLER)
 DEFINE_DMA_IRQ_HANDLER(2, 5, DMA2_CH5_HANDLER)
 #endif
 
+#define RETURN_TCIF_FLAG(s, d, n) if (s == DMA ## d ## _Channel ## n) return DMA ## d ## _FLAG_TC ## n
+
+uint32_t dmaFlag_IT_TCIF(const DMA_Channel_TypeDef *channel)
```

Comment:
This should be optimized - you can pass `dmaChannelDescriptor_t` as parameter and cache value there (or probably use `flagsShift`)

Reply:
Done. Flagshift would ultimately be better - but I will leave it up to another to clean that up.